src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -147,18 +147,30 @@ <h4 class="panel-title border-bottom-small mt" translate>
     <div ng-if="$ctrl.productData.frequency !== 'NA'">
       <div class="panel panel-default give-modal-panel">
         <h4 class="panel-title border-bottom-small mt0 pt-" translate>
-          Transaction Date for Recurring Gifts
+          Transaction Start Date for Recurring Gifts
         </h4>
         <div class="panel-body pt0">
           <div class="row">
+            <div class="col-sm-3">
+              <div class="form-group">
+                <label translate>Month</label>
+                <div class="form-group">
+                  <select class="form-control form-control-subtle"
+                          ng-model="$ctrl.itemConfig['recurring-start-month']"
+                          ng-options="m.value as m.text for m in $ctrl.possibleTransactionMonths($ctrl.nextDrawDate)"
```

Comment:
@OzzieOrca I'm getting a digest loop on this line, any suggestions?  I tried track by, but that changes the value of the options.